tests/CMakeLists.txt

```diff
@@ -2,6 +2,23 @@
 set(PROJECT_TEST_UNIT_DIR ${CMAKE_CURRENT_SOURCE_DIR}/unit)
 set(PROJECT_TEST_MPI_UNIT_DIR ${CMAKE_CURRENT_SOURCE_DIR}/unit/tests_mpi)
 
+find_package(GTest)
+if (NOT GTest_FOUND)
+  include(FetchContent)
```

Comment:
We can use CMake's `FetchContent` instead of pre-installing GTest in Docker image.
This makes it possible to update GTest version independently in various DARMA projects.